composer.json

```diff
@@ -29,13 +29,13 @@
         "php": "^7.1",
         "laminas/laminas-stdlib": "^3.2.1",
         "laminas/laminas-zendframework-bridge": "^0.4.5 || ^1.0",
-        "phpstan/phpstan-shim": "^0.10.5",
+        "phpstan/phpstan": "^0.12",
```

Comment:
This seems to be already addressed in PR #15 
Is this (Version bump for phpstan) needed for applying the CS or was it committed by accident? To me it seems like this is not necessary here.